apps/meteor/client/sidebar/RoomList/SideBarItemTemplateWithData.tsx

```diff
@@ -170,7 +172,7 @@ function SideBarItemTemplateWithData({
 			actions={actions}
 			menu={
 				!isAnonymous &&
-				!isQueued &&
+				!(isQueued && !isPriorityEnabled) &&
```

Comment:
```suggestion
				!isQueued &&
				isPriorityEnabled &&
```

Reply:
The menu is not exclusive for Omnichannel, I can't hide it only based on the priorities. The current check may seem confusing but it's necessary.